app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update]
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def edit; end
```

Comment:
Optional!
But can be missed, because you used it in before_action, but it is ok to mention it as well!